pvlib/inverter.py

```diff
@@ -393,6 +397,7 @@ def pvwatts(pdc, pdc0, eta_inv_nom=0.96, eta_inv_ref=0.9637):
     pdc_neq_0 = ~np.equal(pdc, 0)
 
     # eta < 0 if zeta < 0.006. power_ac is forced to be >= 0 below. GH 541
+    # differs from Eq. 10 of [1], where parentheses were omitted by mistake
     eta = eta_inv_nom / eta_inv_ref * (
         -0.0162 * zeta - np.divide(0.0059, zeta, out=eta, where=pdc_neq_0)
         + 0.9858)  # noQA: W503
```

Comment:
What's the difference between this code and the equation in the reference?  I can't see it:

![image](https://github.com/pvlib/pvlib-python/assets/57452607/4088aa63-34ed-42f3-af9f-c844bc9c50b0)

Reply:
The parentheses are missing in this copy: https://www.nrel.gov/docs/fy14osti/62641.pdf  Which copy did you open?